recipes/zero

```diff
@@ -0,0 +1,5 @@
+(zero
+ :fetcher git
+ :url "https://gitlab.emacsos.com/sylecn/zero-el"
+ :branch "pkg"
+ )
```

Comment:
Don't put closing parenthesis on a line of their own...

Reply:
fixed that in a new force push.